doc/source/examples/extended_examples/hpc/hpc_ml_ga.rst

```diff
@@ -0,0 +1,447 @@
+series of operations, including selection, crossover, and mutation, to produce new
+generations of solutions. The fittest chromosomes, which best satisfy the problem
+constraints and objectives, are more likely to be selected for reproduction, thus
+gradually improving the overall quality of solutions over time. Genetic algorithms are
+particularly useful for solving problems where traditional methods are impractical or
+inefficient, such as optimization, search, and machine learning tasks. They find
+applications in various fields, including engineering, economics, biology, and computer science.
+
+Problem definition
+==================
+
+In this example, a generic algorithm is used to calculate the force
+required for a double clamped beam to deform a specific amount in its
+center.
+
+The beam model is the same as :ref:`ref_mapdl_beam`.
+It is made of ``BEAM188`` elements which span for 2.2 meters,
+and it fully clamped at both ends.
+
+The PyMAPDL beam model is defined in ``calculate_beam`` as follows:
+
+.. code-block:: python
+
+    def calculate_beam(mapdl, force):
+        # Initializing
+        mapdl.clear()
+        mapdl.prep7()
+
+        # Define an I-beam
+        mapdl.et(1, "BEAM188")
+        mapdl.keyopt(1, 4, 1)  # transverse shear stress output
+
+        # Material properties
+        mapdl.mp("EX", 1, 2e7)  # N/cm2
+        mapdl.mp("PRXY", 1, 0.27)  # Poisson's ratio
+
+        # Beam properties in centimeters
+        sec_num = 1
+        mapdl.sectype(sec_num, "BEAM", "I", "ISection", 3)
+        mapdl.secoffset("CENT")
+        mapdl.secdata(15, 15, 29, 2, 2, 1)  # dimensions are in centimeters
+
+        # Setting FEM model
+        mapdl.n(1, 0, 0, 0)
+        mapdl.n(12, 110, 0, 0)
+        mapdl.n(23, 220, 0, 0)
+        mapdl.fill(1, 12, 10)
+        mapdl.fill(12, 23, 10)
+
+        for node in mapdl.mesh.nnum[:-1]:
+            mapdl.e(node, node + 1)
+
+        # Define the boundary conditions
+        # Allow movement only in the X and Z direction
+        for const in ["UX", "UY", "ROTX", "ROTZ"]:
+            mapdl.d("all", const)
+
+        # constrain just nodes 1 and 23 in the Z direction
+        mapdl.d(1, "UZ")
+        mapdl.d(23, "UZ")
+
+        # apply a -Z force at node 12
+        mapdl.f(12, "FZ", force[0])
+
+        # run the static analysis
+        mapdl.run("/solu")
+        mapdl.antype("static")
+        mapdl.solve()
+
+        # Extracting data
+        UZ = mapdl.post_processing.nodal_displacement("Z")
+        UZ_node_12 = UZ[12]
+
+        return UZ_node_12
+
+As it can be seen, this function returns the control parameter for this model, which is the displacement at the Z-direction on the node 12 (``UZ_node_12``).
+
+
+Setting MAPDL pool
+==================
+
+The genetic algorithms are expensive regarding the amount of calculation needed to reach an optimal solution. As seen before, many simulations must be performed to select, crossover, and mutate
```

Comment:
```suggestion
Genetic algorithms are expensive in regard to the amount of calculations needed to reach an optimal solution. As shown earlier, many simulations must be performed to select, cross over, and mutate
```
Two words for cross over as a verb, one word as a verb.

Reply:
I guess you mean "one word as noun"?